### PR TITLE
ROSTESTS-274. TcpIpConnect: Fix a harmless typo

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Trivial code fix. No functional changes.

JIRA issue: [ROSTESTS-274](https://jira.reactos.org/browse/ROSTESTS-274)

## Proposed changes

- 1 s/ok_bool_true/ok/, overkill though harmless.
- 1 s/ntohs/htons/, typo though harmless.

### CODE REVIEW

File: modules/rostests/kmtests/tcpip/tcpip.h [hunk 1]
```diff
@@ -3,5 +3,7 @@
 #define IOCTL_TEST_CONNECT  2
 
 /* For the TDI_CONNECT test */
+// 12345 = 0x3039
 #define TEST_CONNECT_SERVER_PORT 12345
+// 54321 = 0xD431
```
[the remark this answers is not in the document]
Comment (SergeGautherie) [reply modules/rostests/kmtests/tcpip/tcpip.h]: A clue about where ROSTESTS-268 failing value comes from, at least.

[the remark this answers is not in the document]
Comment (sanchaez) [reply modules/rostests/kmtests/tcpip/tcpip.h]: That's really unnesesary.

[the remark this answers is not in the document]
Comment (SergeGautherie) [reply modules/rostests/kmtests/tcpip/tcpip.h]: Removed.

File: modules/rostests/kmtests/tcpip/TcpIp_user.c [hunk 2]
```diff
@@ -57,7 +57,7 @@ AcceptProc(
     ok(Error == 0, "");
 
     ListenSocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
-    ok_bool_true(ListenSocket != INVALID_SOCKET, "socket failed");
+    ok(ListenSocket != INVALID_SOCKET, "socket failed");
```
Comment (ThFabba) [inline modules/rostests/kmtests/tcpip/TcpIp_user.c]: This is now missing its \n

Comment (ThFabba) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: .. which by the way is a perfect example of why you shouldn't make unnecessary changes. If a trivial two line change can introduce a regression, think about what useless refactoring can do.

Comment (sanchaez) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: Ouch.

Comment (sanchaez) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: @ThFabba I will revert it then.

Comment (ThFabba) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: The change is fine, you could just add the \n instead.

Comment (sanchaez) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: Eh, no `git-revert` practice then 😋

Comment (ThFabba) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: :D
Actually there are tons more \n-less ok()s in this file. You could fix those too if you wanted.

Comment (sanchaez) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: I guess I'll leave that to @SergeGautherie 😄

Comment (SergeGautherie) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: > you shouldn't make unnecessary changes

I know. I try to, though I do not always agree. (I don't like stalled "wrong" code.)
I shall take part of the blame for this regression.

> there are tons more \n-less ok()s in this file

I would fix the whole file.
Nonetheless, in this very case, I would rather say the kmtest harness is inconsistent and error-prone, obviously.
Then, I would rather (soon) fix the latter: to add a `\n` in `KmtOk(), KmtTrace() and KmtSkip()`.

Comment (SergeGautherie) [reply modules/rostests/kmtests/tcpip/TcpIp_user.c]: > I would rather (soon) fix the latter

See #349.

**Pull request merged**